### PR TITLE
Refactor Card component (added buttons)

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
Added Props to display edit button when card is in view state, and save, cancel buttons when card is in edit view

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -18,13 +18,14 @@
     display: block;
     font-size: $h3-title-font-size;
     overflow: hidden;
-    padding: $base-spacing $double-spacing;
+    // padding: $base-spacing $double-spacing;
```
Comment (karthik-sista) [inline src/styles/card.scss]: Can you clean this code, by removing the comments?

File: src/styles/card.scss [hunk 2]
```diff
@@ -18,13 +18,14 @@
...
 
     span {
       float: left;
       font-weight: $normal-weight;
-      margin: 0;
-      padding: 0;
+      // margin: 0;
```
Comment (karthik-sista) [inline src/styles/card.scss]: Same thing here.

File: src/styles/card.scss [hunk 3]
```diff
@@ -18,13 +18,11 @@
...
     font-size: $h3-title-font-size;
     overflow: hidden;
-    padding: $base-spacing $double-spacing;
 
     span {
       float: left;
       font-weight: $normal-weight;
-      margin: 0;
-      padding: 0;
+      padding: 1.5rem 3rem;
```
Comment (mlipardo) [inline src/styles/card.scss]: Variables in _constants.scss should be used.

File: src/Cards.js [hunk 4]
```diff
@@ -21,19 +21,28 @@ const Cards = props => {
...
+  props.cardActionButtons ? (editClass = 'edit') : (editClass = '');
   return (
-    <div className={classField}>
+    <div className={classField} id={props.id}>
       <div className={`card ${editClass} double-gap-top`}>
         <div className="card-header">
           <span>{props.cardHeaderText}</span>
+          {props.cardHeaderButton && !props.cardActionButtons ? (
+            <Button btnClassName="default pull-right" btnName="Edit" />
+          ) : (
```
Comment (edgibbs) [inline src/Cards.js]: These 2 just need test coverage.

**Comment by edgibbs** [comment]
@alyssaburkecwds approved

**Pull request merged**